common/utils/src/main/resources/error/error-classes.json

```diff
@@ -846,6 +846,11 @@
       "Exceeds char/varchar type length limitation: <limit>."
     ]
   },
+  "EXCEED_RETRY_JVM" : {
+    "message" : [
+      "Retries exceeded but no exception caught."
+    ]
+  },
```

Comment:
FYI, you will need to run this
```
  /* Used to regenerate the error class file. Run:
   {{{
      SPARK_GENERATE_GOLDEN_FILES=1 build/sbt \
        "core/testOnly *SparkThrowableSuite -- -t \"Error classes are correctly formatted\""
   }}}

   To regenerate the error class document. Run:
   {{{
      SPARK_GENERATE_GOLDEN_FILES=1 build/sbt \
        "core/testOnly *SparkThrowableSuite -- -t \"Error classes match with document\""
   }}}
   */
```
or SparkThrowableSuite will fail.
Remember to git add the file it will generate (if it will... I'm not sure if every class generates a new file; check with git status)